apps/core/src/components/root-app-sidebar/index.tsx

```diff
@@ -175,9 +175,11 @@ export const RootAppSidebar = ({
         }
       >
         <SidebarContainer>
-          <Popover
-            open={openUserWorkspaceList}
-            content={
+          <Menu
+            rootOptions={{
+              open: openUserWorkspaceList,
+            }}
+            items={
               <Suspense>
```

Comment:
I’ve added a skeleton to the workspace card, do you reckon this would enhance the user experience?

https://github.com/toeverything/AFFiNE/assets/102217452/87e7ffcd-a576-4f6a-b22f-ea97239c5592